Userland/Utilities/rm.cpp

```diff
@@ -37,6 +39,12 @@ int main(int argc, char** argv)
 
     bool had_errors = false;
     for (auto& path : paths) {
+        if (not no_preserve_root)
+            if (path == "/") {
+                warnln("rm: '/' is protected, try with --no-preserve-root to override this behavior");
+                continue;
+            }
+
```

Comment:
2 small issues:
* We don't use the `not`/`and`/`or` keywords in Serenity, we prefer `!`/`&&`/`||`
* ifs that contain more than 2 lines (like the outer one) should use curly braces.

Reply:
Noted!